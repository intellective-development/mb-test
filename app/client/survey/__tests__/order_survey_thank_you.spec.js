import * as React from 'react';
import OrderSurveyThankYou from '../order_survey_thank_you';

it('renders', () => {
  expect(render(
    <OrderSurveyThankYou />
  )).toMatchSnapshot();
});
