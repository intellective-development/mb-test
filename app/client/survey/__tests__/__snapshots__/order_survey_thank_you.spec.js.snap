// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<div>
  <div
    class="checkout-frame no-bottom-margin"
  >
    <div
      class="modal-header mega-header"
    >
      <h3>
        Thank you for your feedback
      </h3>
    </div>
  </div>
  <div
    class="checkout-frame no-bottom-margin"
  >
    <div
      class="modal-header mega-header"
    >
      <h3>
        Share with friends
      </h3>
      <h2>
        Get free drinks for spreading the word about Minibar Delivery.
      </h2>
    </div>
    <div
      class="checkout-panel mega-body"
    >
      <div
        class="row"
      >
        <div
          class="large-5 medium-5 push-1 column"
        >
          <p
            class="p-large large-10"
          >
            Give $, get $, when they use your referral code on their first order.
          </p>
        </div>
        <div
          class="large-5 medium-5 pull-1 column"
        >
          <ul
            class="actions"
          >
            <li>
              <span
                class="code-block"
              />
            </li>
            <li>
              <a
                class="button-v2 icon icon-twitter twitter"
                href="https://twitter.com/home?status=Get%20$undefined%20off%20wine,%20spirits%20and%20beer%20on%20your%20first%20@minibardelivery%20order.%20Use%20code:%20undefined%20https://minibardelivery.com"
                rel="noopener noreferrer"
                target="_blank"
              >
                Tweet it out
              </a>
            </li>
            <li>
              <a
                class="button-v2 icon icon-facebook facebook"
                href="https://www.facebook.com/sharer/sharer.php?u=https://minibardelivery.com"
                rel="noopener noreferrer"
                target="_blank"
              >
                Share on Facebook
              </a>
            </li>
            <li>
              <a
                class="button-v2 icon icon-email"
                href="mailto:?Subject=Save%20%24undefined%20on%20your%20first%20order%20from%20Minibar&Body=Hi%2C%0A%0AHere%27s%20a%20%24undefined%20promo%20code%20for%20Minibar%2C%20the%20app%20that%20provides%20wine%2C%20spirits%20and%20beer%20delivered%20to%20your%20door%20in%20under%20an%20hour.%0A%0AMy%20personal%20code%20is%3A%20undefined.%0A%0ATo%20use%20it%2C%20download%20Minibar%20Delivery%20from%20the%20app%20store%20here%3A%20http%3A//mbar.me/MNBjnv%20or%20visit%20the%20website%20minibardelivery.com.%0A%0AI%20get%20a%20%24undefined%20credit%20when%20you%20use%20my%20code%20so%20let%27s%20both%20enjoy%20a%20drink%21%0A%0ACheers"
                rel="noopener noreferrer"
                target="_blank"
              >
                Email friends
              </a>
            </li>
          </ul>
        </div>
      </div>
    </div>
  </div>
  <div
    class="checkout-frame no-bottom-margin"
  >
    <div
      class="modal-header mega-header"
    >
      <h2>
        Go to 
        <a
          href="/"
        >
          minibardelivery.com
        </a>
      </h2>
    </div>
  </div>
</div>
`;
